openaerostruct/aerodynamics/pg_wind_rotation.py

```diff
@@ -166,10 +166,10 @@ def compute(self, inputs, outputs):
         alpha = inputs["alpha"]
         beta = inputs["beta"]
 
-        cosa = np.cos(alpha)
-        sina = np.sin(alpha)
-        cosb = np.cos(beta)
-        sinb = np.sin(beta)
+        cosa = np.cos(alpha)[0]

```

Comment:
A better implementation here would be not to fetch these as scalars, and manually write the rotation matrix. Instead, use an existing implementation such as [scipy.spatial.transform](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.html). Could be done later though, I just feel this is kind of silly

Reply:
That makes sense, I don't want to mess with their implementation too much because I haven't worked with these components much. I'll make the switch to `np.asscalar()` (these will always be of size 1 because the input shape is defined as that size).

Reply:
Yeah this is fine, just something to keep in mind for the future.